package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "eslint-config-smartthings",
-	"version": "3.0.0",
+	"version": "3.3.0",
```

Comment:
Why the jump from 3.0.0 to 3.3.0?

Reply:
I like to update to .3 . Want to revert it back for .1 ?

Reply:
Unless it's conflicting with published version, versioning should follow semver.
https://semver.org/

Reply:
Changed to .1 fo version